olive/evaluator/olive_evaluator.py

```diff
@@ -428,6 +428,7 @@ def _evaluate_distributed_accuracy_worker(config) -> Tuple[List[Any], List[Any]]
         local_rank = config["local_rank"]
         world_size = config["world_size"]
         inference_settings = config.get("inference_settings", {}) or {}
+        execution_providers = config.get("providers", ["CUDAExecutionProvider"]) or ["CUDAExecutionProvider"]
```

Comment:
>or ["CUDAExecutionProvider"] [](http://example.com/codeflow?start=81&length=28)

I am supposing this is dead-code since the dict.get will return the default value specified in the second parameter?

Reply:
Resolved.